Pipfile

```
@@ -9,7 +9,7 @@ watchdog = "*"
 cryptography = "*"
 pynacl = "*"
 redis = "*"
-tuf = "==2.0.0"
+tuf = "*"
 dynaconf = {extras = ["ini"], version = "*"}
 supervisor = "*"
 securesystemslib = "*"
```

I believe at least in `repository_service_tuf_worker/services/keyvault/local.py` (`decrypt_key`, `encrypt_key`) we use the cryptography feature so we might want to install the `crypto` optional dependency:
```suggestion
securesystemslib = {extras = ["crypto"], version = "*"}
```

---

What will help us to have `crypto` as an optional dependency?

---

We already use `pynacl` https://github.com/secure-systems-lab/securesystemslib#installation
Not sure if we need crypto. 🤔

---

My comment was based on https://github.com/vmware/repository-service-tuf-cli/issues/131 which we also required crypto when installing `securesystemslib`